test/e2e-smoke-tests.sh

```diff
@@ -47,18 +47,16 @@ header "Running smoke tests"
 kubectl create ns $KN_E2E_SMOKE_TESTS_NAMESPACE || fail_test
 
 ./kn service create hello --image gcr.io/knative-samples/helloworld-go -e TARGET=Knative -n $KN_E2E_SMOKE_TESTS_NAMESPACE || fail_test
-sleep 5
```

Comment:
this should probably be separated into a different PR.

Reply:
sure, raised https://github.com/knative/client/pull/220